apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -132,45 +137,51 @@ const styles = {
   description: {
     paddingRight: 10,
     paddingBottom: 10,
-    fontSize: 16,
-    fontFamily: 'Gotham 3r',
-    zIndex: 2,
+    fontSize: 14,
+    fontFamily: '"Gotham 5r"',
     color: color.charcoal,
-    width: 940
   }
 };
 
-const NewProjectButtons = React.createClass({
-  propTypes: {
+const TILES_PER_ROW = 4;
+
+class NewProjectButtons extends React.Component {
+  static propTypes = {
     projectTypes: PropTypes.arrayOf(PropTypes.string),
-    isRtl: PropTypes.bool
-  },
+    isRtl: PropTypes.bool,
+    description: PropTypes.string,
+  };
 
   render() {
-    const { isRtl } = this.props;
+    const { isRtl, description } = this.props;
     const thumbnailStyle = isRtl ? styles.thumbnailRtl : styles.thumbnail;
     // Using absolute URLs to get this working in storybook.
     const projectTypes = this.props.projectTypes || DEFAULT_PROJECT_TYPES;
     return (
       <div style={styles.fullsize}>
-        <div style={styles.description}>{i18n.projectStartNew()}</div>
-        <div>
-          {
-            projectTypes.slice(0,4).map((projectType, index) => (
-              <a key={index} href={"/projects/" + projectType + "/new"}>
-                <div style={[styles.tile, index < 3 && styles.tilePadding]}>
-                  <img style={thumbnailStyle} src={PROJECT_INFO[projectType].thumbnail} />
-                  <div style={styles.label}>
-                    {PROJECT_INFO[projectType].label}
-                  </div>
-                </div>
-              </a>
-            ))
-          }
-        </div>
+        {description && <div style={styles.description}>{description}</div>}
+        {
+          _.chunk(projectTypes, TILES_PER_ROW).map((projectTypesRow, rowIndex) => (
```

Comment:
Does this still work with <4 projects? Maybe we should add that to the storybook. The project widget can be rendered with 1 - many types.

Reply:
Yes, the story for Project Widget shows several `NewProjectButtons`s, some of which have less than 4 projects, and those are still looking good.

Reply:
Awesome - sorry I missed that